<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BalancePrintBase</class>
 <widget class="QWidget" name="BalancePrintBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Impresion del balance</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QToolButton" name="mui_centroCostes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../bulmages.qrc">
         <normaloff>:/BulmaCont32x32/images/png/i_centroCoste.xpm</normaloff>:/BulmaCont32x32/images/png/i_centroCoste.xpm</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="mui_canales">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../bulmages.qrc">
         <normaloff>:/BulmaCont32x32/images/png/i_canales.xpm</normaloff>:/BulmaCont32x32/images/png/i_canales.xpm</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>430</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Datos del listado:</string>
     </property>
     <layout class="QVBoxLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="margin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="labelCuentaInicial">
            <property name="text">
             <string>Cuenta inicial:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlSearchWidget" name="mui_cuentaInicial"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="labelFechaInicial">
            <property name="text">
             <string>Fecha inicial</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlDateSearch" name="mui_fechaInicial"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="labelCuentaFinal">
            <property name="text">
             <string>Cuenta final:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlSearchWidget" name="mui_cuentaFinal"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="labelFechaFinal">
            <property name="text">
             <string>Fecha final</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BlDateSearch" name="mui_fechaFinal"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <layout class="QHBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="labelNivel">
            <property name="text">
             <string>Nivel de cuentas:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="mui_nivel"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
          <item>
           <widget class="QCheckBox" name="mui_nivelSuperior">
            <property name="text">
             <string>Incluir &amp;niveles superiores.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mui_parciales">
            <property name="text">
             <string>Incl&amp;uir resultados parciales</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Formato:</string>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
        <item>
         <widget class="QRadioButton" name="mui_pdf">
          <property name="text">
           <string>&amp;PDF</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mui_textoPlano">
          <property name="text">
           <string>&amp;Texto Plano</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="mui_html">
          <property name="text">
           <string>HT&amp;ML</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_generaImpresion">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Imprimir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="5"/>
 <customwidgets>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QWidget</extends>
   <header>bldatesearch.h</header>
  </customwidget>
  <customwidget>
   <class>BlSearchWidget</class>
   <extends>QWidget</extends>
   <header>blsearchwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc"/>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
